src/constants/poi.data.ts

```diff
@@ -0,0 +1,1463 @@
+import { POI, POICategory, BuildingId, FloorOverlay } from "../types/main";
+
+export const floorOverlays: FloorOverlay[] = [
+  // Test floor, will be deleted
+  {
+    id: 1,
+    buildingId: BuildingId.H,
+    level: 1,
+    bounds: [
+      [45.49772, -73.579507],
+      [45.49679, -73.578391]
+    ],
+    image: require("../../assets/floors/H-1.png")
+  },
+  {
+    id: 2,
+    buildingId: BuildingId.MB,
+    level: 1,
+    bounds: [
+      [45.4956, -73.57953],
+      [45.494916, -73.57846]
+    ],
+    image: require("../../assets/floors/MB-1.png")
+  }
+];
+
+export const POIInfo: POI[] = [
+  // ------------------
+  // LEVEL 8
+  // ------------------
+  {
+    id: "",
```

Comment:
The ids should be updated in the future

Reply:
Indeed, this pr mainly serves to unify the models and types, the actual values will be updated in the related us story.

Reply:
For consistency, I will set id's to be numbers. and still also fix a hidden error with the data I just noticed that